### DISCUSSION TITLE
So, i have some questions ⬇️

### DISCUSSION THREAD

**Discussion by ghost** [Atlas-OS/Atlas#1166] [Help & Support] [answered]
I've noticed people telling me that for each atlas update, you need to reinstall normal windows and then install atlas over again?
and that i myself noticed i cannot change the start menu icon, which i want to set my own picture, is that possible?

**Comment by Ast3risk-ops** [comment]
Not. For. All. Updates.

0.5.0 for instance won't require a reinstall *if you are updating from 0.4*.

**Reply by ghost** [reply]
> Not. For. All. Updates.
> 
> 0.5.0 for instance won't require a reinstall _if you are updating from 0.4_.

i mean it's been a long time since i used atlas, currently on stock 24h2
and another thing is, why can't i change startmenu icon

**Reply by Ast3risk-ops** [reply]
> > Not. For. All. Updates.
> > 0.5.0 for instance won't require a reinstall _if you are updating from 0.4_.
> 
> i mean it's been a long time since i used atlas, currently on stock 24h2 and another thing is, why can't i change startmenu icon

Windows has never let you do that ever, you'll need another program to do that (google "change start menu icon").

**Reply by ghost** [reply]
> > > Not. For. All. Updates.
> > > 0.5.0 for instance won't require a reinstall _if you are updating from 0.4_.
> > 
> > 
> > i mean it's been a long time since i used atlas, currently on stock 24h2 and another thing is, why can't i change startmenu icon
> 
> Windows has never let you do that ever, you'll need another program to do that (google "change start menu icon").

previously atlasos included third-party start menu, which i couldn't change the atlas logo to my own

**Comment by he3als** [comment, marked as answer]
> and that i myself noticed i cannot change the start menu icon, which i want to set my own picture, is that possible?

Do you mean your user account profile picture? If so, you should be able to change it. If you're having issues, check for Windows Updates first, as there was previously a bug in Windows 11 that didn't allow users with a local account to change it.

> I've noticed people telling me that for each atlas update, you need to reinstall normal windows and then install atlas over again?

To upgrade to v0.4 from an older Atlas version, a full reinstallation of Windows using our installation guide is necessary. However, when v0.5.0 releases, v0.4 users will be able to upgrade to it without a reinstall.
